.vsts-ci/phase.yml

```diff
@@ -35,10 +35,6 @@ jobs:
     - pwsh: ' az login --service-principal -u $(az_url) -p $(az_key) --tenant $(az_name)'
       displayName: 'az login'
 
-  - pwsh: Install-module pester -Scope CurrentUser -Force
```

Comment:
Removed so we can test the solution

Reply:
I think it was a good idea to test it, but can we put it back.  I  like the logs separate.  It's easier to diagnose if something goes wrong.